app/models/katello/sync_plan.rb

```diff
@@ -23,13 +23,20 @@ class SyncPlan < Katello::Model
     validates :interval, :inclusion => {:in => TYPES}, :allow_blank => false
     validates :enabled, :inclusion => [true, false]
     validate :validate_sync_date
+    validate :product_enabled
     validates_with Validators::KatelloNameFormatValidator, :attributes => :name
 
     scoped_search :on => :name, :complete_value => true
     scoped_search :on => :organization_id, :complete_value => true, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
     scoped_search :on => :interval, :complete_value => true
     scoped_search :on => :enabled, :complete_value => true
 
+    def product_enabled
+      products.each do |product|
+        errors.add :product, _("#{product.name} can not be disabled.") unless product.enabled?
```

Comment:
With the :product error key, ActiveModel::Errors will prepend that symbol as a string to whatever error messages you have; Just keep that in mind. It's idiomatic to use the attribute that contains the error (products in this case), but I'd be fine with using :base. @ehelms what do you think?